matomo/templates/cronjob-ipdb.yaml

```
@@ -0,0 +1,52 @@
+{{- if .Values.ipdb.enabled -}}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "matomo.fullname" . }}-ipdb
+  labels: {{- include "matomo.labels" . | nindent 4 }}
+spec:
+  schedule: {{ .Values.ipdb.schedule | quote }}
+  concurrencyPolicy: {{ .Values.ipdb.concurrencyPolicy | default "Allow" }}
+{{- if .Values.ipdb.startingDeadlineSeconds }}
+  startingDeadlineSeconds: {{ .Values.ipdb.startingDeadlineSeconds }}
+{{- end }}
+  successfulJobsHistoryLimit: {{ .Values.ipdb.successfulJobsHistoryLimit | default 1 }}
+  failedJobsHistoryLimit: {{ .Values.ipdb.failedJobsHistoryLimit | default 1 }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+        {{- if .Values.image.pullSecrets }}
+          imagePullSecrets:
+          {{- range .Values.image.pullSecrets }}
+            - name: {{ . }}
+          {{- end }}
+        {{- end }}
+          restartPolicy: Never
+          containers:
+          - name: {{ include "matomo.name" . }}
+            image: "{{ .Values.ipdb.image.repository }}:{{ .Values.ipdb.image.tag }}"
+            imagePullPolicy: {{ .Values.ipdb.image.pullPolicy }}
```

Der Gewinn durch das Image ist zu gering um hier noch eine Version zu managen.
```suggestion
            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
            imagePullPolicy: {{ .Values.image.pullPolicy }}
```